test/t8n/t8n.cpp

```diff
@@ -230,11 +230,13 @@ int main(int argc, const char* argv[])
 
             if (!pre_state_only && rev >= EVMC_PRAGUE)
             {
-                requests.emplace_back(collect_deposit_requests(receipts));
-
-                auto system_call_requests =
-                    system_call_block_end(state, block, block_hashes, rev, vm);
-                std::ranges::move(system_call_requests, std::back_inserter(requests));
+                // FIXME: Report invalid block if system contracts execution fails.
```

Comment:
if you don't intend to fix in this PR or immediately after consider renaming to a TODO